academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -171,37 +197,288 @@ The input entities file Url prefix.
 
 The output url prefix for writing the built index.
 
-### Optional Parameters
+### BuildIndex optional parameters
+
+`--SchemaUrl`
+
+The url to the input index schema definition file. By default, the url is set to the schema describing the latest MAKES index release.
+
+`--SynonymResourceFolderUrl`
+
+The url to a resource folder containing all synonym data files required by the input index schema.
 
 `--IndexPartitionCount`
 
 The number of index partitions to create. A index job can finish quicker when there are more partitions; however, the more partitions, the less accurate the interpret results will be. Maximize build performance by creating 1 partition per worker.
 
-`--IntersectionMinCount`
+`--IntersectionCountThresholdForPreCompute`
 
-The minimum intersections between indexed attribute for which the indexer should generate pre-calculated results. The higher the count, the more process will be required at run time. The lower the count, the larger the generated index will be.
+The attribute value intersections threshold for pre-computing look up tables. Use this value to tune index build time performance, run time performance, and index size. The higher the value, the smaller index size and slower run-time performance will be. The lower the value, the larger index size and faster run-time performance will be.
```

Comment:
Should we describe more of what the attribute value intersections are?

Reply:
We'd need a new section explaining the concept of materialized intersections for this to make more sense. I'm not confident illustrating that part of KES yet..

---
In reply to: [465189345](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/172#discussion_r465189345) [](ancestors = 465189345)